modules/vaos/config/routes.rb

```diff
@@ -9,6 +9,7 @@
       resources :messages, only: %i[index create]
     end
     get 'community_care/eligibility/:service_type', to: 'cc_eligibility#show'
+    get 'community_care/supported_sites', to: 'cc_supported_sites#index'
```

Comment:
I'm still not sure you want this to be an index action -- we're not returning a collection of results. If we had a unique id for each member, it would be an index. Instead we're getting a constant back that just happens to be an array. But we can revisit this once cassettes are recorded.

Reply:
If site-code could be thought of as an "id" then we could build the route such that site_code is part of the url, not a query string. I think this would make more sense, but then you'd have a "show" (member action with site_code as id) not an "index" (which returns a collection).

Reply:
site_code is a comma separated list of site ids, so not sure if that changes your mind one way or another.

Reply:
hmmm, this doesn't seem particularly RESTful to me to be honest. I guess if site_code is thought of as an array of parameters, then perhaps index makes the most sense.

Reply:
cool, looks like we kept it as index -- resolved